app/controllers/account/dashboard_controller.rb

```diff
@@ -13,6 +13,18 @@ def user_info_card
 
   end
 
+  def user_info
+    array_of_dates = (6.days.ago.to_date..Date.today).map{ |date| date }
```

Comment:
same comment as for Tanya ... move logic from controller to class (speak with Tanya)